src/test/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/uiTest/pages/SettingsFrame.kt

```diff
@@ -0,0 +1,165 @@
+package nl.tudelft.ewi.se.ciselab.testgenie.uiTest.pages
```

Comment:
Are these basically libraries for common IntelliJ UI elements in the settings?

Reply:
These are not libraries, but files defined by me :P

Reply:
ah, yes 🥇